dnf-behave-tests/dnf/plugins-core/leaves.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Show packages not needed by any package
```

Comment:
We usually have two empty lines after the `Feature` line, could you add those to be consistent?